machine/v1beta1/types_gcpprovider.go

```diff
@@ -134,6 +156,16 @@ type GCPKMSKeyReference struct {
 	Location string `json:"location"`
 }
 
+// GCPAcceleratorConfig describes type and count of accelerator cards attached to the instance on GCP.
+type GCPAcceleratorConfig struct {
+	// AcceleratorCount is number of AcceleratorType accelerators (GPUs) to be attached to an instance
+	AcceleratorCount int32 `json:"acceleratorCount"`
+	// AcceleratorType is the type of accelerator (GPU) to be attached to an instance.
+	// Supported accelerator types are: nvidia-tesla-k80, nvidia-tesla-p100, nvidia-tesla-v100, nvidia-tesla-p4, nvidia-tesla-t4
+	// +kubebuilder:validation:Pattern=`^nvidia-tesla-(k80|p100|v100|p4|t4)$`
+	AcceleratorType string `json:"acceleratorType"`
+}
+
```

Comment:
I think we should rename this inline with the rename above or remove the accelerator prefixes as below
```suggestion
// GCPGPUConfig describes type and count of accelerator cards attached to the instance on GCP.
type GCPGPUConfig struct {
	// Count is number of GPUs to be attached to an instance
	Count int32 `json:"count"`
	// Type is the type of GPU to be attached to an instance.
	// Supported accelerator types are: nvidia-tesla-k80, nvidia-tesla-p100, nvidia-tesla-v100, nvidia-tesla-p4, nvidia-tesla-t4
	// +kubebuilder:validation:Pattern=`^nvidia-tesla-(k80|p100|v100|p4|t4)$`
	Type string `json:"type"`
}

```